test/models/dto/AddressDtoSpec.scala

```diff
@@ -340,12 +336,9 @@ class AddressDtoSpec extends BaseSpec {
 
       AddressDto.ukForm.bind(formData).fold(
         formWithErrors => {
-          formWithErrors.errors.length shouldBe 2
-          formWithErrors.errors(0).message shouldBe "error.line3_required"
-          formWithErrors.errors(1).message shouldBe "error.line4_required"
         },
         success => {
-          fail("Form should give an error")
+          success shouldBe AddressDto("Line 1", "Line 2", None, None, Option("Line 5"), Some("AA1 1AA"), None, None)
```

Comment:
```suggestion
          success shouldBe AddressDto("Line 1", "Line 2", None, None, Some("Line 5"), Some("AA1 1AA"), None, None)
```

Reply:
Added fail

Reply:
Thanks but this comment was about using `Some` instead of `Option`